src/lib.rs

```diff
@@ -90,6 +87,8 @@ fn send_with_fd(socket: RawFd, bs: &[u8], fds: &[RawFd]) -> io::Result<usize> {
         ptr::write(
             cmsg_header,
             libc::cmsghdr {
+                #[cfg(target_env = "musl")]
```

Comment:
Would it perhaps make more sense to construct `cmsghdr` by assigning fields to a `zeroed` value the same way, it is done for `msghdr` above?

Reply:
done